modules/reference/pages/software/channel-details.adoc

```diff
@@ -1,97 +1,70 @@
 [[s4-sm-channel-details-details]]
 = Channel Details
 
-General information about the channel and its parent if applicable.
-This summary, description, and architecture is also displayed when clicking a channel.
+The menu:Software[Channel List > Channel Details] section displays the details of a selected patch.
```

Comment:
I'm wondering whether such menu paths are ok.  It is kind of an abbreviation. You cannot click a widget with the name "Channel Details".
```suggestion
The menu:Software[Channel List > Channel Details] section displays the details of a selected channel.
```
copypasta? :wink:

Reply:
yup